integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/apim/utils/AdminUtils.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.tests.apim.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.am.integration.clients.admin.api.AdvancedPolicyCollectionApi;
+import org.wso2.am.integration.clients.admin.api.ApplicationPolicyCollectionApi;
+import org.wso2.am.integration.clients.admin.api.SubscriptionPolicyCollectionApi;
+import org.wso2.am.integration.clients.admin.api.dto.*;
+import org.wso2.am.integration.test.Constants;
+import org.wso2.am.integration.test.impl.RestAPIAdminImpl;
+import org.wso2.choreo.connect.tests.context.CCTestException;
+
+import java.util.List;
+
+public class AdminUtils {
+
+    /**
+     * Get all Advanced Throttling Policies from Admin Portal.
+     *
+     * @param adminRestClient an instance of RestAPIAdminImpl
+     * @return list of all Advanced Throttling Policies
+     * @throws CCTestException if an error occurs while retrieving Advanced Throttling Policies
+     */
+    public static List<AdvancedThrottlePolicyInfoDTO> getAllAdvancedThrottlingPolicies(RestAPIAdminImpl adminRestClient)
+            throws CCTestException {
+        // Currently, RestAPIAdminImpl does not have a method to get all AdvancedThrottlingPolicies.
+        // This is a workaround to use a method that is used internally by RestAPIAdminImpl.
+        AdvancedPolicyCollectionApi advancedPolicyCollectionApi = new AdvancedPolicyCollectionApi();
+        advancedPolicyCollectionApi.setApiClient(adminRestClient.apiAdminClient);
+        AdvancedThrottlePolicyListDTO advancedThrottlePolicyListDTO;
+        try {
+            advancedThrottlePolicyListDTO = advancedPolicyCollectionApi
+                    .throttlingPoliciesAdvancedGet(Constants.APPLICATION_JSON, null, null);
+        } catch (org.wso2.am.integration.clients.admin.ApiException e) {
```

Comment:
We can import this at the start.

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/2736/commits/7048d5c96997f9537de523dba580cdd561454894